sound/soc/sof/pm.c

```diff
@@ -215,9 +215,13 @@ static int sof_resume(struct device *dev)
 	if (!sdev->ops->resume)
 		return 0;
 
+	/* reset the previous flag of pm process */
+	sdev->dsp_error_pending = false;
```

Comment:
this flag is only cleared after firmware successful boot.

Reply:
ok, I will move it to sof_pcm_open() function.
overthere the firmware will be ok.